aws-toolkit-eclipse-master/bundles/com.amazonaws.eclipse.sdk.ui/samples/AmazonEC2SpotInstances-Advanced/InlineTaggingCodeSampleApp.java

```diff
@@ -93,9 +93,9 @@ public static void main(String[] args) {
         }
 
         // Create the AmazonEC2Client object so we can call various APIs.
-        AmazonEC2 ec2 = AmazonEC2ClientBuilder.standard()
-            .withCredentials(new AWSStaticCredentialsProvider(credentials))
-            .withRegion("us-west-2")
+        AmazonEC2 ec2 = AmazonEC2ClientBuilder.standard()

```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

AWS Region is set using a `String`. To explicitly set a publicly available region, we recommend that you use the [Regions](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/index.html?com/amazonaws/regions/Regions.html) enum.

[Learn more](https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/java-dg-region-selection.html#region-selection-choose-region)